libr/anal/p/anal_arc.c

```diff
@@ -1092,6 +1092,6 @@ struct r_anal_plugin_t r_anal_plugin_arc = {
 struct r_lib_struct_t radare_plugin = {
 	.type = R_LIB_TYPE_ANAL,
 	.data = &r_anal_plugin_arc,
-	.version = R2_VERSION
+	.version = R2_VERSION,
```

Comment:
I added that comma so that the sys/indent works properly - without it, the indent rules yank the closing brace up onto that line instead.